ml-agents/mlagents/trainers/ppo/policy.py

```diff
@@ -253,9 +251,7 @@ def get_value_estimates(
         if self.use_vec_obs:
             feed_dict[self.model.vector_in] = [brain_info.vector_observations[idx]]
         if self.use_recurrent:
-            if brain_info.memories.shape[1] == 0:
-                brain_info.memories = self.make_empty_memory(len(brain_info.agents))
-            feed_dict[self.model.memory_in] = [brain_info.memories[idx]]
+            feed_dict[self.model.memory_in] = self.retrieve_memories([idx])
```

Comment:
Does this need to be `[self.retrieve_memories([idx])]` to match the old behavior? Then again, the old behavior looks weird...

Reply:
No, retrieve memories will always return a 2d array while brain_info.memories returned a 1d one